nyc/templates/base.html

```diff
@@ -58,7 +58,7 @@
                         <a href='{% url 'events' %}'>{{ CITY_VOCAB.EVENTS }}</a>
                     </li>
                     <li>
-                        <a href="{% url 'nyc:search' %}">Legislation</a>
+                        <a href="{% url 'nyc:search' %}?sort_by=relevance">Legislation</a>
                     </li>
```

Comment:
We do not want to do this: it does not address the issue and also undermines [our default sorting criteria (i.e., by date)](https://github.com/datamade/nyc-council-councilmatic/blob/master/nyc/views.py#L237).

The problem in the issue concerns the ordering of results, _after_ searching for a keyword. So, if I search for "Hearing Transcripts 2002 2015-3716" - it should be the first thing that we return.